src/integrationTest/java/com/mongodb/spark/sql/connector/write/MongoSparkConnectorWriteTest.java

```diff
@@ -98,13 +98,13 @@ void testSupportedWriteModesWithOptions() {
         df.write()
             .format("mongodb")
             .mode("Overwrite")
-            .option(MongoConfig.PREFIX + MongoConfig.COLLECTION_NAME_CONFIG, "coll2");
+            .option(MongoConfig.COLLECTION_NAME_CONFIG, "coll2");
 
     dfw.save();
     assertEquals(0, getCollection().countDocuments());
     assertEquals(10, getCollection("coll2").countDocuments());
 
-    dfw.option(MongoConfig.WRITE_PREFIX + MongoConfig.COLLECTION_NAME_CONFIG, "coll3").save();
+    dfw.option(MongoConfig.COLLECTION_NAME_CONFIG, "coll3").save();
```

Comment:
This is the user facing impact of the options are scoped so the user can use the short form `dfw.option("collection", "coll3")` instead of `dfw.option("spark.mongodb.write.collection")`